app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -527,17 +527,28 @@ class MigrationsProvider(val context: Context) {
         }
     }
 
-    // todo: This is VPN project migration, KEEP IT ALWAYS LAST
     val MIGRATION_40_TO_41: Migration = object : Migration(40, 41) {
```

Comment:
@aitorvs can you double check this code as I removed the call to MIGRATION_39_TO_40 (and all comments). I tested a migration from 5.101.0 to latest and worked, however I didn't have AppTP enabled (if it mattered).